lib/builder.js

```diff
@@ -39,7 +39,7 @@ exports.createZipFile = async function (directory) {
   }
 
   const imageId = await execa.stdout('docker', ['build', '--quiet', '--file', 'scandium-dockerfile', '.'], { cwd: directory })
-  const zipFile = await execa.stdout('docker', ['run', '--rm', imageId], { cwd: directory, maxBuffer: 50000000 })
+  const zipFile = await execa.stdout('docker', ['run', '--rm', imageId], { cwd: directory, maxBuffer: 50000000 * 2 })
```

Comment:
🤔

Reply:
>Should we allow to build larger zips than 50 MB so you can debug if it's getting to large?


I'm thinking maybe allowing the buffer here to be 100 MB and then In the lambda upload check the size and not allow over 50 MB there? maybe even automatically save the file if it's larger than 50 MB to /tmp so you can inspect it?

Reply:
Set it to `100000000` then 😄, `* 2` looks very odd?